### DISCUSSION TITLE
the source code of `Unsqueeze Elimination`

### DISCUSSION THREAD

**Discussion by KotomiHacker** [microsoft/onnxruntime#16285] [Other Q&A] [answered]
Could you help point out where the source code of `Unsqueeze Elimination` is

**Comment by edgchen1** [comment, marked as answer]
https://github.com/microsoft/onnxruntime/blob/d1e8d4a2613f25823d997a38d51b138ca9adcedb/onnxruntime/core/optimizer/unsqueeze_elimination.cc
